Mathlib/Data/Nat/Defs.lean

```diff
@@ -1377,6 +1377,12 @@ lemma dvd_div_of_mul_dvd (h : a * b ∣ c) : b ∣ c / a :=
   ⟨fun h => mul_dvd_of_dvd_div hbc h, fun h => dvd_div_of_mul_dvd h⟩
 #align nat.dvd_div_iff Nat.dvd_div_iff
 
+@[simp] lemma div_dvd_iff_dvd_mul (h : b ∣ a) (hbpos : 0 < b) : a / b ∣ c ↔ a ∣ b * c :=
```

Comment:
I think the forward direction is true without `hbpos`; can you prove it separately?

Reply:
Done!